Assignment2/alvaro3420-AlvaroToledo.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF xml:base="http://www.ontogrid.net/StickyNote#"
+    xmlns="http://www.ontogrid.net/StickyNote#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+    <rdf:Description rdf:about="#Class01">
+	<includes rdf:resource="#Sensor029"/>
+	<includes rdf:resource="#Computer101"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+		<hasMeasurement rdf:resource="#Measurement8401"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Measurement8401"/>
+		<hasTemperature rdf:datatype="http://www.w3.org/2001/XMLSchema#integer">29</hasTemperature>
+		<atTime rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2010-06-12T12:00:12</atTime>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Computer101">
+		<hasOwner rdf:resource="#User10A"/>
+    </rdf:Description>
+    <rdf:Description rdf:about="#User10A">
+		<hasName rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Pedro</hasName>
+    </rdf:Description>
+    <rdf:Description rdf:about="#Sensor029">
+    </rdf:Description>
+    <rdf:Description rdf:about="#Computer101">
+    </rdf:Description>
+    <rdf:Description rdf:about="#Measurement8401">
+    </rdf:Description>
+	<rdf:Description rdf:about="#User10A">
+    </rdf:Description>
+	<rdf:Description rdf:about="#Pedro">
```

Comment:
This is not needed, and indeed wrong as this resource does not exist. Anyway, I will accept it now.